substrafl/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.45.0"
+__version__ = "0.46.0"
```

Comment:
Would not:
```suggestion
__version__ = "0.45.1"
```
Be more appropriate?

Reply:
Mmmmh it's a minor change so it could enter in the minor definition of semver.

Reply:
I would argue that this is more a security patch transparent for the user than it is a new functionality but this is no big deal

Reply:
Yes I get what you mean... I'm arguing against changing the version il all PRs ahah 😇

Reply:
Fair point, that's why it has been approved beforehand 😃